### PR TITLE
Delete animation on stop

### PR DESCRIPTION
**Pull request by azigerelli**
Adds functionality to the `AnimationProvider` so that animations are deleted after they are stopped. Also found and fixed another bug. I noticed if an animation was active and you click the restart button and then cycle through the scenes back to where you were, the animation would start as soon the scene loaded. I added a function to clear all animations from the registry and call that any time restart is pressed.

### CODE REVIEW

**Review by ajd6735 — APPROVED** [review]

**Review by zlix12 — COMMENTED** [review]
Looks good logically. Will approve after merge conflicts are resolved.

File: src/artifacts/Orbiter.tsx
Comment (zlix12) [inline src/artifacts/Orbiter.tsx:1]: Can we reset this file to main version?

**Review by zlix12 — CHANGES_REQUESTED** [review]

**Review by zlix12 — APPROVED** [review]
LGTM

**Pull request merged**
